src/index.js

```diff
@@ -14,12 +14,14 @@ export default (
     restage = true,
     branch,
     bail,
+    check,
     verbose,
     onFoundSinceRevision,
     onFoundChangedFiles,
     onPartiallyStagedFile,
+    onProcessFile,
+    onCheckFile,
     onWriteFile,
-    onExamineFile,
```

Comment:
Sorry to be a pain, but we can prevent this from being an API breaking change by keeping it called `onExamineFile` instead of `onProcessFile`. Not sure if anyone uses this module's API, but doesn't hurt to not break it.

Reply:
No problem! So you're thinking in the public API (i.e., just `index.js`), this would remain `onExamineFile`, but internally it would be called `onProcessFile` (per https://github.com/azz/pretty-quick/pull/64#discussion_r284496821)? Or would you prefer to call these `onExamineFile`, `onWriteFile`, and `onCheckFile` across the board?

Reply:
The latter

Reply:
Done!